frontend/dockerfile/parser/testfiles/trailing-backslash/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# https://github.com/docker/for-win/issues/5254
+
+FROM hello-world
+
+ENV A path
+ENV B another\\path
+ENV C trailing\\backslash\\
+ENV D This gets appended to C
```

Comment:
@thaJeztah this is a bit confusing as the text says it gets appended to C, but it doesn't and it shouldn't, right?

Reply:
Good one, yes, I copied the "reproducer" example, but I guess it would be better to change to "This should not be appended to C" in the test